### PR TITLE
Update Appx cmdlets for WS2022

### PR DESCRIPTION
**Pull request by PatAltimore**
[PR Review comment](https://github.com/MicrosoftDocs/windows-powershell-docs/pull/2563#issuecomment-919366442)

### CODE REVIEW

**Comment by PatAltimore** [comment]
#sign-off

File: docset/winserver2022-ps/appx/Add-AppxPackage.md
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/appx/Add-AppxPackage.md:493]:
```suggestion
- UsePreference: Uses the current [PackageSubPreference](/uwp/api/windows.management.deployment.packagestubpreference) for the package
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @PatAltimore, kindly commit the suggested change. Many thanks.

**Comment by PatAltimore** [comment]
Hi @JohanFreelancer9,

Thanks for the fix. I committed your suggestion.

Thanks,
Pat

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by PatAltimore** [comment]
Hi @tiburd,

I got one approval so far on this. Could you help it along? It can be merged after approval.

Thanks,
Pat

**Review by dariomws — APPROVED** [review]
Minor suggestion.

File: docset/winserver2022-ps/appx/Add-AppxPackage.md
Comment (get-itips) [inline docset/winserver2022-ps/appx/Add-AppxPackage.md:142]: Shouldn't this link be `/windows/msix/app-installer/how-to-create-appinstaller-file` ?

Comment (JohanFreelancer9) [reply docset/winserver2022-ps/appx/Add-AppxPackage.md:142]: @get-itips - nice catch. Agree.

Comment (JohanFreelancer9) [inline docset/winserver2022-ps/appx/Add-AppxPackage.md:142]:
```suggestion
For more information, see [Create an App Installer file manually](/windows/msix/app-installer/how-to-create-appinstaller-file).
```

Comment (dariomws) [reply docset/winserver2022-ps/appx/Add-AppxPackage.md:142]: Although it's tempting, do not shorten links in cmdlet reference topics. Although the shortened links will work on the web, when the topics are converted into XML and checked into code for help at the command line, the shortened links will not work; only the full and complete URL will work.
Because of this, you'll notice that there's no rule in the office-docs-powershell repo suggesting shortened links.
Resource:
https://github.com/MicrosoftDocs/office-docs-powershell/pull/8231#issuecomment-903925410

Should it apply to this repo?

**Review by get-itips — APPROVED** [review]

File: docset/winserver2022-ps/appx/Add-AppxPackage.md
Comment (PatAltimore) [reply docset/winserver2022-ps/appx/Add-AppxPackage.md]: Thanks for the feedback. 

Hi @dariomws, @JohanFreelancer9 is right. If the article exists in docs.microsoft.com, use a relative link. You can make the link relative by removing the https://docs.microsoft.com from the beginning.

I've made the change. Thanks for the help.

Comment (PatAltimore) [reply docset/winserver2022-ps/appx/Add-AppxPackage.md]: Thanks for the feedback. I made the link relative.

**Review by tseward — APPROVED** [review]

**Comment by v-dihans** [comment]
Hi @PatAltimore  

**Following are blocking issues that need to be resolved before we can merge:**

docset/winserver2022-ps/appx/Get-NonRemovableAppsPolicy.md
- Lines 159-162: Add a valid language identifier to the code block.


***

**Reference**

[Criteria used for pull request reviews](https://review.docs.microsoft.com/en-us/office-authoring-guide/pr-criteria?branch=master)

**Blocking**
- Code blocks in an article must have a [valid language identifier](https://docsmetadatatool.azurewebsites.net/allowlists#) applied.

***

Thanks

**Comment by PatAltimore** [comment]
Hi @v-dihans,

Could you clarify the required change more? I looked at line number range in the article. It's not a code block. It's a bulleted list of allowed values for the parameter. Is that the right location?

Thanks,
Pat

**Comment by PatAltimore** [comment]
Hi @v-dihans,

Could you clarify the change? Or if it's okay, could you approve the PR?

Thanks,
Pat

**Comment by v-dihans** [comment]
@PatAltimore  

Apologies Pat, I did not notice your previous comment.  

The issue is that the information on those lines is displaying as a code block:
image

I think if you back out the indent a little bit, it will fix the problem.

Thanks

**Comment by PatAltimore** [comment]
Hi @v-dihans,

Thanks for the clarification. It looks like removing the indent fixed it.

Thanks,
Pat

**Comment by v-dihans** [comment]
@PatAltimore  

Thanks for the fix, Pat! Glad that worked for you.

For future, whenever you want us to re-review a PR, please add the Sign-off label, and remove the Back to submitter label. That way, the PR enters the queue again and we know to review it.

Cheers!

**Pull request merged**
